hooks-admin.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+// Filter export formats array
+function pb_mpdf_add_to_formats( $formats ) {
+		unset( $formats['standard']['pdf'] );
```

Comment:
Forcing an unset of formats is prescriptive, is there a reason to keep lines 5 and 6?